packages/block-library/src/latest-comments/edit.js

```diff
@@ -75,6 +75,7 @@ export default function LatestComments( { attributes, setAttributes } ) {
 				<ServerSideRender
 					block="core/latest-comments"
 					attributes={ attributes }
+					urlQueryArgs={ { _locale: '' } }
```

Comment:
Prefer to add comment to make it clear for the reason why set the locale to empty string because IMO it's not intuitive to know.